src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue

```diff
@@ -147,6 +157,7 @@ export default {
 		// now it breaks validation. Another option: Create NcNumberField component
 		setAmount(value) {
 			this.amount = parseFloat(value)
+			this.isInvalidAmount = this.amount > 20 || this.amount < 1
```

Comment:
```suggestion
			this.isInvalidAmount = isNaN(this.amount) || !this.$refs.textField.$refs.inputField.$refs.input?.checkValidity()
```
Maybe a bit ugly, but it checks, if input is not empty and not out of range of provided `min`/`max`

Reply:
Literally the same validation as the current one ?

Reply:
You just got lucky, that `NaN < 1` =)
It's safer, and you define max/min ranges only at the one place (component attrs)